servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/Versions.groovy

```diff
@@ -21,6 +21,7 @@ import static org.gradle.api.JavaVersion.VERSION_1_8
 
 final class Versions {
   static final String CHECKSTYLE_VERSION = "8.41.1"
+  static final String PITEST_VERSION = "1.6.0"
```

Comment:
This constant is never used, consider removing it

Reply:
Correct. The PITest plugin doesn't seem to support the toolVersion property.